packages/tools/kadena-cli/src/tx/commands/txTestSignedTransaction.ts

```diff
@@ -61,7 +61,7 @@ export const createTestSignedTransactionCommand: (
   program: Command,
   version: string,
 ) => void = createCommandFlexible(
-  'test-signed-transaction',
+  'test',
   'Test a signed transaction on testnet.',
```

Comment:
Should we have an alias for this `local` as everyone knows about the `/local` call?

Reply:
i dont follow?